Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml

```diff
@@ -6475,85 +6475,285 @@ script:
+      required: false
+      secret: false
+    - default: false
+      description: The status of a VPC peering connection, if applicable.
+      isArray: false
       name: UserIdGroupPairsPeeringStatus
-    - description: 'The ID of an AWS account. For a referenced security group in another
+      required: false
+      secret: false
+    - default: false
+      description: The ID of an AWS account. For a referenced security group in another
         VPC, the account ID of the referenced security group is returned in the response.
         If the referenced security group is deleted, this value is not returned. [EC2-Classic]
         Required when adding or removing rules that reference a security group in
-        another AWS account.'
+        another AWS account.
+      isArray: false
       name: UserIdGroupPairsUserId
-    - description: The ID of the VPC for the referenced security group, if applicable.
+      required: false
+      secret: false
+    - default: false
+      description: The ID of the VPC for the referenced security group, if applicable.
+      isArray: false
       name: UserIdGroupPairsVpcId
-    - description: The ID of the VPC peering connection, if applicable.
+      required: false
+      secret: false
+    - default: false
+      description: The ID of the VPC peering connection, if applicable.
+      isArray: false
       name: UserIdGroupPairsVpcPeeringConnectionId
-    - description: The AWS Region, if not specified the default region will be used.
+      required: false
+      secret: false
+    - default: false
+      description: The AWS Region, if not specified the default region will be used.
+      isArray: false
       name: region
-    - description: The Amazon Resource Name (ARN) of the role to assume.
+      required: false
+      secret: false
+    - default: false
+      description: The Amazon Resource Name (ARN) of the role to assume.
+      isArray: false
       name: roleArn
-    - description: An identifier for the assumed role session.
+      required: false
+      secret: false
+    - default: false
+      description: An identifier for the assumed role session.
+      isArray: false
       name: roleSessionName
-    - description: The duration, in seconds, of the role session. The value can range
+      required: false
+      secret: false
+    - default: false
+      description: The duration, in seconds, of the role session. The value can range
         from 900 seconds (15 minutes) up to the maximum session duration setting for
         the role.
+      isArray: false
       name: roleSessionDuration
+      required: false
+      secret: false
+    deprecated: false
     description: Removes egress rule from a security group. To remove a rule, the
       values that you specify (for example, ports) must match the existing rule's
       values exactly.
     execution: true
     name: aws-ec2-revoke-security-group-egress-rule
-  dockerimage: demisto/boto3:2.0.0.18824
+  - arguments:
```

Comment:
arent you missing here the client args (region, roleArn, etc..)?

Reply:
I'll add them, but I didn't add this command

Reply:
arent those the arguments of aws-ec2-allocate-hosts ?

Reply:
yes sorry, got confused with the command above. 
Added them